evm/contracts-deployment-test.js

```diff
@@ -8,19 +8,48 @@ const {
 } = ethers;
 const { Command, Option } = require('commander');
 
-const { mainProcessor } = require('./utils');
+const { mainProcessor, saveConfig } = require('./utils');
 const { addBaseOptions } = require('./cli-utils');
 
-async function processCommand(_, chain, options) {
+async function processCommand(config, chain, options) {
     const wallet = new Wallet(options.privateKey, new JsonRpcProvider(chain.rpc));
 
+    if (
+        !config.chains[options.chainNames].contracts.Multisig ||
+        !config.chains[options.chainNames].contracts.Multisig.signers ||
+        !config.chains[options.chainNames].contracts.Multisig.threshold
+    ) {
+        config.chains[options.chainNames].contracts.Multisig = { signers: [wallet.address], threshold: 1 };
```

Comment:
add support for parsing `--args` for these contracts to avoid having to do this manipulation. could simplify our onboarding commands as well